src/domain/collaboration/CalloutBlock/CalloutLayout.tsx

```diff
@@ -110,7 +110,7 @@ const CalloutLayout = ({
       return undefined;
     }
 
-    return t(`common.enums.callout-state.${state}` as const);
```

Comment:
it's weird, but I have noticed that `common.enums.callout-state` is not used anywhere else